packages/graphics/spirv-llvm-translator/package.mk

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2024-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="spirv-llvm-translator"
+PKG_VERSION="18.1.0"
+PKG_SHA256="78a770eff24d5ffe2798479845adec4b909cbf058ddc55830ea00fa7d2c1698a"
+PKG_LICENSE="Apache-2.0"
```

Comment:
are you sure ? looks like another licence clusterfuck looking at the licence.txt at the git

Reply:
updated to included License file.